dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "backstage-community-plugin-redhat-argocd",
+  "version": "1.6.6",
```

Comment:
released an hour ago... https://www.npmjs.com/package/@backstage-community/plugin-redhat-argocd/v/1.6.7

Even said so in this PR:

https://github.com/backstage/community-plugins/pull/1112#issuecomment-2315986580

Please wait until 1.6.8 is out and rebase against that version